drools-game-engine-services/src/main/java/org/drools/game/services/endpoint/api/GameService.java

```diff
@@ -63,5 +68,25 @@
     @Consumes( value = APPLICATION_JSON )
     @Produces( value = APPLICATION_JSON )
     void destroy( String sessionId );
+    
+    @POST
+    @Path( "{sessionId}/execute" )
+    @Consumes( value = APPLICATION_JSON )
+    @Produces( value = APPLICATION_JSON )
+    <T> void execute( @PathParam( "sessionId" ) String sessionId, CommandRest<T> cmdRest );
+
+    
+    @GET
+    @Path(  "{sessionId}/player/{playerName}" )
+    @Consumes( value = APPLICATION_JSON )
+    @Produces( value = APPLICATION_JSON )
+	Player getPlayer( @PathParam( "sessionId" ) String sessionId,  @PathParam( "playerName" ) String playerName );
+
+    @GET
+    @Path(  "{sessionId}/item/{itemName}" )
+    @Consumes( value = APPLICATION_JSON )
+    @Produces( value = APPLICATION_JSON )
+	Zone getItem( @PathParam( "sessionId" ) String sessionId, @PathParam( "itemName" ) String itemName);
```

Comment:
I would change this method for something like "executeComandQuery (String sessionId, CommandQuery cmdq)", that allow us retract any object from the session.